utils/stringUtils.js

```diff
@@ -104,7 +105,8 @@ const getTribunalPanelWelsh = ben => {
     carersAllowance: 'barnwr',
     attendanceAllowance: 'barnwr, meddyg ac arbenigwr anabledd',
     bereavementBenefit: 'barnwr',
-    UC: 'barnwr ac, ar gyfer rhai apeliadau, meddyg'
+    UC: 'barnwr ac, ar gyfer rhai apeliadau, meddyg',
+    industrialInjuriesDisablement: 'barnwr a hyd at 2 feddyg ymgynghorol'
```

Comment:
Can you add a test for this new scenario(s) in StringUtils.test.js?

Reply:
Done